recipes/pytorch_tabular/build_base.sh

```diff
@@ -0,0 +1 @@
+${PYTHON} -m pip install . --no-deps -vv
```

Comment:
We need this in multiple outputs recipe to avoid a conda bug where these default settings are ignored.

```suggestion
$PYTHON -m pip install --no-deps --no-build-isolation -vv .
```